prov/verbs/src/verbs_eq.c

```diff
@@ -540,6 +540,8 @@ vrb_eq_xrc_connreq_event(struct vrb_eq *eq, struct fi_eq_cm_entry *entry,
 send_reject:
 	if (rdma_reject(connreq->id, *priv_data, *priv_datalen))
 		VRB_WARN(FI_LOG_EP_CTRL, "rdma_reject %d\n", -errno);
+	if (rdma_destroy_id(connreq->id))
```

Comment:
Need to reset connreq->id.  I don't know XRC code but it seems we need to reset ep->tgt_id and maybe other info as well on reject.  i will leave this for others to comment.

Reply:
Yes, also need to reset `ep->tgt_id` since it is referencing a destroyed id, and `ep->base_ep.info_attr.handle`.  Not 100% sure, but `ep->recip_req_received` may need to be reset as well.